### PR TITLE
Feat/about navigation

### PR DESCRIPTION
**Pull request by Fedeya**
**About**

image

**Navigation**
image

### CODE REVIEW

File: src/components/AboutSection/index.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  <section
+    id="about"
+    className="flex relative pt-10 flex-wrap pb-5 w-full px-5 items-center mt-24 mb-12"
+    style={{ backgroundColor: '#F5F9FF' }}
+  >
+    <div
+      className="bottom-auto top-0 left-0 right-0 w-full absolute pointer-events-none overflow-hidden -mt-20"
+      style={{ height: '80px', transform: 'translateZ(0)' }}
+    >
+      <svg
```
Comment (marbiano) [inline src/components/AboutSection/index.tsx:13]: What is this inline SVG?

File: public/nav-logo.svg [hunk 2]
```diff
@@ -0,0 +1,32 @@
+<svg width="172" height="29" viewBox="0 0 172 29" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (marbiano) [inline public/nav-logo.svg:1]: Did you make this SVG file to go through https://jakearchibald.github.io/svgomg/?

**Review by marbiano — APPROVED** [review]
@Fedeya great work! Minor requests, feel free to merge as soon as those have been handled.

File: src/components/AboutSection/index.tsx (hunk 1 above)
Comment (Fedeya) [reply src/components/AboutSection/index.tsx:13]: @marbiano the waves

File: public/nav-logo.svg (hunk 2 above)
Comment (Fedeya) [reply public/nav-logo.svg:1]: @marbiano this svg is directly getted of figma, i should use that website? what is the difference?

**Pull request merged**